src/node_sqlite.h

```diff
@@ -317,6 +318,26 @@ class SQLTagStore : public BaseObject {
   friend class StatementExecutionHelper;
 };
 
+class AuthorizerFunction {
+ public:
+  AuthorizerFunction(Environment* env,
+                     v8::Local<v8::Function> fn,
+                     DatabaseSync* db);
+  ~AuthorizerFunction();
+  static int xAuthorizer(void* user_data,
+                         int action_code,
+                         const char* param1,
+                         const char* param2,
+                         const char* param3,
+                         const char* param4);
+  static void xDestroy(void* self);
```

Comment:
I assume the `x` is to mark these methods as static? We don't typically do this. I realize `UserDefinedFunction` does the same thing, but I'd say it should be removed there too

Reply:
The original intention in those other examples would have been to correspond to the corresponding parameter name from the sqlite3 API call ([example](https://www.sqlite.org/c3ref/create_function.html)), although that doesn't apply here.

Reply:
Oh, I see ... yeah, feel free to go on this either way then, although I'd mildly prefer to just sticking to regular Node.js coding styles